internal/pkg/attestationpolicy/attestationpolicy.go

```diff
@@ -65,13 +65,23 @@ func (ap *AttestationPolicy) GetHelmConfig(source cofidectl_plugin.DataSource, b
 	return clusterSPIFFEID, nil
 }
 
-func getAPLabelSelectorHelmConfig(selector *attestation_policy_proto.APLabelSelector) map[string]interface{} {
+func getAPLabelSelectorHelmConfig(selector *attestation_policy_proto.APLabelSelector) map[string]any {
```

Comment:
I've opted to use `map[string]any` instead of `map[string]interface{}` to aid readability. I think we can update any additional references of `map[string]interface{}` not related to these changes, in a follow up PR.